api/scpca_portal/templates/readme/metadata_only.md

```diff
@@ -0,0 +1,56 @@
+
+Generated on: {{ date }}
+
+# Alex's Lemonade Stand Foundation Single-cell Pediatric Cancer Atlas
+
+The [Single-cell Pediatric Cancer Atlas](https://scpca.alexslemonade.org) is a database of single-cell and single-nuclei data from pediatric cancer clinical samples and xenografts, built by the [Childhood Cancer Data Lab](https://www.ccdatalab.org/) at [Alex's Lemonade Stand Foundation](https://www.alexslemonade.org/).
+
+## Contents
+
+{% if project_accession %}
```

Comment:
we shouldn't pass `project_accession` here we should expect:

`included_projects` - a list of tuples where the first item is the accession and the second is the url

We should instead check that the `included_projects` is greater than 1 and display the "else" content here
If it is only 1 just iterate over the tuples.